### ISSUE TITLE
failed to install volcano

### ISSUE THREAD

**Issue by hwdef** [volcano-sh/volcano#1432]
**What happened**:

when I run commond 

```shell
kubectl apply -f https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml
```

I received two errors.

![image](https://user-images.githubusercontent.com/13084946/115674642-72143380-a380-11eb-89d5-0d78ee245e01.png)

**What you expected to happen**:

Successful installation

**How to reproduce it (as minimally and precisely as possible)**:

```shell
kubectl apply -f https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml
```

**Anything else we need to know?**:

![image](https://user-images.githubusercontent.com/13084946/115674762-9243f280-a380-11eb-8580-34b74ab58ae9.png)

**Environment**:
- Volcano Version: latest
- Kubernetes version (use `kubectl version`): 1.21.0
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by Thor-wl** [comment]
Please downgrade your k8s cluster version to v1.18

**Comment by shinytang6** [comment]
You can also refer to https://github.com/volcano-sh/volcano/issues/1413, and use helm to have a try

**Comment by hwdef** [comment]
@Thor-wl 
emmm,,volcano only supports up to  v1.18?

**Comment by hwdef** [comment]
@shinytang6 
ok, thank you.

**Comment by Thor-wl** [comment]
> @Thor-wl
> emmm,,volcano only supports up to v1.18?

> @Thor-wl
> emmm,,volcano only supports up to v1.18?

It is on the way to support v1.19

**Comment by hwdef** [comment]
@Thor-wl 
Is there an approximate time point for 1.20 and 1.21?

**Comment by Thor-wl** [comment]
> @Thor-wl
> Is there an approximate time point for 1.20 and 1.21?

Actually, we will support the latest three stable version.

**Comment by hwdef** [comment]
@Thor-wl  ok, thank you!

**Comment by hwdef** [comment]
@Thor-wl By the way, is there an approximate timeline?

**Comment by hwdef** [comment]
@Thor-wl 

I noticed that support for 1.19 has been merged, but I still get this error with 1.21.

**Comment by Thor-wl** [comment]
> @Thor-wl
> 
> I noticed that support for 1.19 has been merged, but I still get this error with 1.21.

Since v1.21 is the newest version(there is just v1.21.0) and we will not support it immediately.
